### ISSUE TITLE
Oh, this is getting better every day

### ISSUE THREAD

**Issue by zerthimon** [raelgc/scudcloud#553]
latest release
I've been loosing login to a team recently. https://github.com/raelgc/scudcloud/issues/549
Now I've started loosing entire teams configurations.

Think I'm about done with this client.

**Comment by raelgc** [comment]
Hi @zerthimon. I can realize your frustration and I really apologize about all these inconveniences.

Until the previous release, ScudCloud was using Qt4, which was a way stable, but the webkit (the HTML engine) included on it was too old, and I was introducing several CSS and JS workarounds to try to keep ScudCloud working.

Additionally, I was forced to release before the planned schedule the Qt5 version for Ubuntu 16.10 and Arch, as they drop support to Qt4.

Then, after several months working in the Qt5 version (keep in mind this is, basically, an one man effort working with no documentation), I released as 1.50 (current version).

I was able to remove all the CSS/JS workarounds, but the underlying code (for Network, etc) from Qt5 is not so stable, mainly under slow connections (which I was not expecting).

I'm trying to catchup with all the latest raised issues, but it's not an easy task.

I can understand your frustration. Fortunately, the official client is really good and works pretty well.

Again, sorry, hope we can fix all these issues soon.
